reflect.go

```diff
@@ -0,0 +1,456 @@
+package main
+
+import (
+	"fmt"
+	"go/types"
+	"path/filepath"
+
+	"golang.org/x/exp/maps"
+	"golang.org/x/exp/slices"
+	"golang.org/x/tools/go/ssa"
+)
+
+var checkedAPIs = make(map[string]bool)
```

Comment:
Maybe a variable should be part of `transformer`?

Reply:
with a slightly better name ideally, like `reflectCheckedAPIs`